README.md

```diff
@@ -1,31 +1,43 @@
+<img src="https://raw.githubusercontent.com/hexojs/awesome-hexo/master/hexo-logo.png" alt="Hexo logo" width="100" height="100" align="right" />
+
 # Hexo
 
-[![Build Status](https://travis-ci.org/hexojs/hexo.svg?branch=master)](https://travis-ci.org/hexojs/hexo)
+> A fast, simple & powerful blog framework, powered by [Node.js](https://nodejs.org).
+
+[Website](https://hexo.io) |
+[Documentation](https://hexo.io/docs/) |
+[Installation Guide](https://hexo.io/docs/#Installation) |
+[Contribution Guide](https://hexo.io/docs/contributing) |
+[API](https://hexo.io/api/) |
+[Twitter](https://twitter.com/hexojs)
+
 [![NPM version](https://badge.fury.io/js/hexo.svg)](https://www.npmjs.com/package/hexo)
-[![Coverage Status](https://coveralls.io/repos/hexojs/hexo/badge.svg?branch=master)](https://coveralls.io/r/hexojs/hexo?branch=master)
+![Required Node version](https://img.shields.io/node/v/hexo)
+[![Build Status](https://travis-ci.org/hexojs/hexo.svg?branch=master)](https://travis-ci.org/hexojs/hexo)
 [![Build status](https://ci.appveyor.com/api/projects/status/hpx3lduqjj2t6uqq/branch/master?svg=true)](https://ci.appveyor.com/project/tommy351/hexo/branch/master)
```

Comment:
also suggest to update the badge source to `https://ci.appveyor.com/api/projects/status/github/hexojs/hexo?branch=master&svg=true`, which is more readable.

Too bad, the appveyor link is still tommy351/hexo, not hexojs/hexo.